dali/base/dasds.cpp

```diff
@@ -2778,12 +2787,26 @@ class CNodeSubscriptionManager : public CSimpleInterface, implements INodeSubscr
         CriticalBlock b(lock);
         _notify(&node, state);
     }
-    void notifyDelete(CServerRemoteTree &node)
+    void notifyDelete(CServerRemoteTree *node)
     {
         // shouldn't be here, unless node is in subscribers table
-        CriticalBlock b(lock);
-        _notify(&node, PDS_Deleted);
-        _removeNode(&node, 0);
+        {
+            CriticalBlock b(lock);
+            /* Need to be careful not to release subscribers here (on this thread)
```

Comment:
minor: This comment is slightly misleading.  The problem is with releasing subscribers before _notify is called.